pkg/apis/kops/componentconfig.go

```diff
@@ -655,8 +655,11 @@ type CloudConfiguration struct {
 	NodeInstancePrefix *string `json:"nodeInstancePrefix,omitempty"`
 
 	// vSphere cloud-config specs
+	VSphereUsername      *string `json:"vSphereUsername,omitempty"`
```

Comment:
hmm seems you have run conversion-gen yet? If you run conversion-gen, a few files like zz_generated.conversion.go should be updated too.